pkg/apis/pipeline/v1alpha1/taskrun_validation_test.go

```diff
@@ -0,0 +1,326 @@
+package v1alpha1
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/knative/pkg/apis"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestTaskRunValidate(t *testing.T) {
+	tests := []struct {
+		name string
+		task TaskRun
+		want *apis.FieldError
```

Comment:
should we have a function with a success case too? e.g. `TestTaskRunValidate` and `TestTaskRunValidate_Invalid`

Reply:
thanks!!